src/Inkluzitron/Models/ValuesDisposingDictionary.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Inkluzitron.Models
+{
+    public class ValuesDisposingDictionary<TKey, TValue> : Dictionary<TKey, TValue>, IDisposable
```

Comment:
Ten název se mi zdá trochu weird. Spíš bych to pojmenoval `DisposableDictionary`.

Reply:
Právě mi přišlo dobré uvést v názvu explicitně to, že se disposují jen hodnoty, ne klíče.